pages/apis/graphql/schemas/agentconnection.md.erb

```diff
@@ -0,0 +1,36 @@
+<!--
+  _____   ____    _   _  ____ _______   ______ _____ _____ _______ 
+  |  __  / __   |  | |/ __ __   __| |  ____|  __ _   _|__   __|
+  | |  | | |  | | |  | | |  | | | |    | |__  | |  | || |    | |   
+  | |  | | |  | | | . ` | |  | | | |    |  __| | |  | || |    | |   
+  | |__| | |__| | | |  | |__| | | |    | |____| |__| || |_   | |   
+  |_____/ ____/  |_| _|____/  |_|    |______|_____/_____|  |_|   
+  This file is auto-generated by script/generate_graphql_api_content.sh,
+  please build the schema.json by running `rails api:graph:export`
```

Comment:
Is this `rake` or `rails`? And what are the dependencies? More than the standard `buildkite/buildkite` ones?